sitesupport/configuration.js

```diff
@@ -115,7 +115,7 @@ var configHeader = function(options) {
 }
 
 
-var settingsTable = function(options) {
+export var settingsTable = function(options) {
```

Comment:
We are not a module yet here... Does this work outside of module code?

Reply:
I haven't seen an issue yet, treating this like a module, but I can do some more digging to make certain. I think the only things that actually use it are already in code that are either deliberately modularized or close enough for jazz, but I'll double-check it.

Reply:
Yeah, digging around, I may have to revert this one for the moment.

Reply:
Ok tell me what you come up with one way or the other.  Feels like reverting the export seems like the right things.